Archane.js

```diff
@@ -0,0 +1,1169 @@
+        console.log(chalk`You roll a {red ${playerDeceptionRoll}} {cyan (Charisma Modifier of ${playerCharismaStatModifier(playerCharisma)})}`)
+        if (playerDeceptionRoll >= 11) {
+            console.log(chalk`\nYou were able to loosen up the rope around you, giving the {magenta impression} that you are still tied up but are able to slip out at any time.`)
+            guardAppearance();
+        } else {
+            console.log(chalk`\nYou try to free yourself from the rope.
+{red You end up giving yourself a cramp in your arm.}
+You also unintentonally made the rope tighter around you.`)
+            guardAppearance();
+        }
+    } else if (playerChairChoices === "stats") {
+        logPlayerStats();
+        playerChairChoice();
+    } else if (playerChairChoice === "inventory") {
+        logplayerInventory();
+        playerChairChoice();
+    } else {
+        playerChairChoice();
+    }
+}
+
+const threeDoors = () => {
+    doors = [chalk.green("First Door"), chalk.yellow("Second Door"), chalk.red("Third Door")]
+    console.log(`Which door do you go through?`)
+    doorChoice = rls.keyInSelect(doors)
+    playerDoorChoice = doors[doorChoice]
+    if (playerDoorChoice === chalk.green("First Door")) {
+        console.log(chalk`\nYou decided to go through the {green first door}.
+
+When you walk in, you see an abundance of books. It's a library.
+On your immediate right there is a table with four books on top. The titles read:
+
+The Curse of The Magi (Vol. {magenta 2})
+Magic for Beginners (Vol. {yellow 8})
+The Best Cookbook for Dragonborns (Vol. {blue 5})
+How To Train Your Pet Human (Vol. {green 3})
+
+You take a look around and realize there is nothing really useful here.
+You go back into the hallway.`)
+        doors.shift()
+        console.log("\nWhich door do you choose?")
+        doorChoice = rls.keyInSelect(doors)
+        playerDoorChoice = doors[doorChoice]
+        if (playerDoorChoice === chalk.yellow("Second Door")) {
+            console.log(chalk`\nYou decided to go through the {yellow second door}.
+When you walk in, you are greeted with the smell of delicious food... as well as a multitude of guards.
+They ended up killing you.
+{red You died.} `)
+            restartGame();
+        } else if (playerDoorChoice === chalk.red("Third Door")) {
+            let numberOfTries = 3
+            if (playerInventory.includes(playerWeapon(playerClass))) {
+                console.log(chalk`\nYou decided to go through the {red third door}.
+You see a chest that requires a code. 
+You try at opening the chest. {cyan (Hint: Pay attention to the volume numbers.)}`)
+                for (i = numberOfTries; i !== 0; i++) {
+                    console.log(chalk`\n{cyan You have {red ${numberOfTries}} tries left.}`)
+                    playerCodeInput = rls.questionInt("What code do you try?: ")
+                    if (playerCodeInput === 2853) {
+                        chestOpen();
```

Comment:
Good practice to break out of loops to ensure we don't accidentally go back after chestOpen() path is complete.